validate.py

```diff
@@ -258,8 +262,9 @@ def update(event, context):
         return config.add_cors_headers({"statusCode":400,"body":"User email not found."})
 
     #validate the updates, passing only the allowable ones through.
-    updates = validate_updates(results, event['updates'], a_res)
+    updates = validate_updates(results, event['updates'], a_res) #the core of possible security holes. make validate_updates stronger.
```

Comment:
This comment is already semi-reflected above. But I get it as a part of personal notes.... can we do away with most of them, please?